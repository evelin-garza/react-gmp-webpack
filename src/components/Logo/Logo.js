import React from "react";

import "./Logo.scss";

const Logo = () => {
    return (
        <span className="logo"><strong>netflix</strong>roulette</span>
    );
};

export default Logo;