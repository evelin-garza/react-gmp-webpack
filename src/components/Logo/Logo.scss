.logo {
    font-size: 20px;
    color: #F65261;

    strong {
        font-size: 20px;
    }
}