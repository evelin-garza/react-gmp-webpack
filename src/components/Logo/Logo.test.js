import { render } from "@testing-library/react";
import React from "react";

import Logo from "./Logo";

describe('Logo Component', () => {
    it('should render logo', () => {
        const { asFragment } = render(<Logo />);

        expect(asFragment()).toMatchSnapshot();
    });
});