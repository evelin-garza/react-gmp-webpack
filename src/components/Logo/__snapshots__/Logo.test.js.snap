// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should render logo 1`] = `
<DocumentFragment>
  <span
    class="logo"
  >
    <strong>
      netflix
    </strong>
    roulette
  </span>
</DocumentFragment>
`;
